src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplyCount = 0;
+        int buyCount = 0;
+
+        //String report = reportPrint(supplyCount, buyCount);
+        // writeToFile(toFileName, report);
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
```

Comment:
we need to put buffered reader into try with resources block as well, to close connection to file after we're done